### PR TITLE
allow for & implement - multiple dropdowns on hover

### PR DESCRIPTION
**Pull request by chrislabarge**
Community Hover Action #856 [here](https://github.com/TheOdinProject/theodinproject/issues/856)

### CODE REVIEW

**Comment by chrislabarge** [comment]
Do I have to include Gemlock file for Travis CI build to pass?

**Comment by mindovermiles262** [comment]
It appears this PR is failing CI because `nokogiri` has a known vulnerability. Please run `bundle update` and then push to your `additional-hover-event-in-nav` branch

**Comment by arun1595** [comment]
@chrislabarge I've merged a [PR](https://github.com/TheOdinProject/theodinproject/pull/862) to bump the nokogiri version. Please update your PR branch with master.

**Comment by chrislabarge** [comment]
Thank you @arun1595 just updated.

File: app/assets/javascripts/navbar.js [hunk 1]
```diff
@@ -3,30 +3,33 @@ document.addEventListener('turbolinks:load', function() {
...
-  if(userDropdown) {
-    userDropdown.addEventListener('mouseover', function() {
-      userDropdown.classList.add('show')
+
+  toggleDropdownsOnHover();
+})
+
+// Open Dropdowns on Hover, hide on mouseout
+function toggleDropdownsOnHover() {
+  var dropdowns = document.querySelectorAll('.js-dropdown')
```
Comment (mindovermiles262) [inline app/assets/javascripts/navbar.js]: let's move this to the top of the file with the others

**Comment by chrislabarge** [comment]
@mindovermiles262 sounds good! All fixed.

**Comment by KevinMulhern** [comment]
Thanks you @chrislabarge

**Pull request merged**